numpy/core/src/multiarray/arrayobject.h

```diff
@@ -40,6 +40,13 @@ static const int NPY_ARRAY_WARN_ON_WRITE = (1 << 31);
 static const int NPY_ARRAY_WAS_PYTHON_INT = (1 << 30);
 static const int NPY_ARRAY_WAS_PYTHON_FLOAT = (1 << 29);
 static const int NPY_ARRAY_WAS_PYTHON_COMPLEX = (1 << 28);
+/*
+ * Mark that this was a huge int and the array needed replace (no re-use).
```

Comment:
replace -> replacement
But maybe
```
Mark that this was a hugh int which was turned into an object array, but then
replaced by a temporary array for further processing. This flag is only in the
ufunc machinery where it is tricky to cover correctly all type resolution paths.
```

Reply:
thanks, adopted with a bracket that it can also be unsigned int (technically, on windows even int64 vs. int32, but I don't want to worry about it).